docs/examples/networking/nat/nat.tf

```diff
@@ -0,0 +1,226 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "compartment_ocid" {}
+variable "region" {}
+variable "ssh_public_key" {}
+
+# Choose an Availability Domain
+variable "AD" {
+    default = "1"
+}
+
+variable "InstanceShape" {
+    default = "VM.Standard1.2"
+}
+
+variable "InstanceOS" {
+    default = "Oracle Linux"
+}
+
+variable "InstanceOSVersion" {
+    default = "7.4"
+}
+
+variable "vcn_cidr" {
+    default = "10.0.0.0/16"
+}
+
+variable "mgmt_subnet_cidr" {
+    default = "10.0.0.0/16"
+}
+
+variable "private_subnet_cidr" {
+    default = "10.0.0.0/16"
```

Comment:
When running using these defaults I'm getting the error below - I think this needs to be a different value from the other subnet:

* oci_core_subnet.PrivateSubnet: Status: 400; Code: InvalidParameter; OPC Request ID: /87AF334F38C76347EE6B866A2F26EA55/943BBE5065234BEE81B795DD3DFC2518; Message: The requested CIDR 10.0.0.0/16 is invalid: subnet ocid1.subnet.oc1.iad.aaaaaaaa4fdarrsa7jyqy7wrs5athfpct55dea2n5cntqypu4grd5fehjjwa with CIDR 10.0.0.0/16 overlaps with this CIDR.